<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>351</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="xmageTab">
    <attribute name="title">
     <string>XMage</string>
    </attribute>
    <widget class="QLabel" name="xmageTableLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Installed XMage Versions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTreeWidget" name="xmageTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>XMage Branch</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>XMage Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="xmageAdd">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xmageRemove">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>220</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="javaTab">
    <attribute name="title">
     <string>Java</string>
    </attribute>
    <widget class="QLabel" name="javaTableLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto-detected Java Installations</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="javaSelection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="javaSelectionLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Selected Java</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="javaBrowse">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>260</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="javaTree">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>181</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Java Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Java Vendor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
